src/tools/asciitab.js

```diff
@@ -2,7 +2,8 @@ const { isChord } = require('markdown-it-music/lib/chord');
 
 const State = {
   DEFAULT: 1,
-  VERSE: 2
+  VERSE: 2,
+  FENCE: 3
```

Comment:
What does this do? Otherwise LGTM

Reply:
During one of our rehearsals, I ran the auto-formatter on a file that had:

```
:::abc
...
:::
```

The auto-formatter detected the `:::` as a verse heading due to [getMaybeHeading](https://github.com/music-markdown/music-markdown/blob/3633ecba8542375948426c829ce661d068b9e050/src/tools/asciitab.js#L9) matching on the line ending with a `:`. The original purpose of that line is to detect things like `Verse 1:` and convert it into `## Verse 1`

The problem was that the auto-formatter doesn't ignore text between fences. The code you commented on adds a state to the parser for fences. When a line starts with `:::`, we ignore all text proceeding that line until we see another line with `:::`. During that time, the state of the parser is `FENCE`.